contrib/pg_tde/expected/partition_table.out

```diff
@@ -52,6 +52,7 @@ SELECT pg_tde_is_encrypted('partition_q4_2024');
 (1 row)
 
 ALTER TABLE partitioned_table SET ACCESS METHOD heap;
+ERROR:  Recursive ALTER TABLE on a mix of encrypted and unencrypted relations is not supported
 ALTER TABLE partition_q1_2024 SET ACCESS METHOD heap;
```

Comment:
Maybe not as part of this ticket, but shouldn't we add a warning when somebody creates mixed encrypted/not encrypted partitions / typed tables?

Reply:
That was a question I was thinking about and I am uncertain. I say let's do/investigate if we should do this as a separate ticket.